tests/meta/upload/conftest.py

```diff
@@ -115,19 +151,37 @@ def import_reads(*args, **kwargs):
 
 
 class MockScoutApi:
-    """ Mock class for Scout api"""
+    """Mock class for Scout api"""
 
     @staticmethod
     def get_causative_variants(case_id):
         """mock get_causative_variants"""
         _ = case_id
         return []
 
+    @property
+    def client(self):
+        """Client URI"""
+        return "mongodb://"
+
+    @property
+    def name(self):
+        """Name property"""
+        return self._name
+
+    @name.setter
+    def name(self, value):
+        self._name = value
+
+    @name.deleter
+    def name(self):
+        del self._name
+
```

Comment:
You wrote that these were added for the linting and I don't understand for which warnings?